http-body-util/src/full.rs

```diff
@@ -18,6 +18,11 @@ impl<D> Full<D>
 where
     D: Buf,
 {
+    /// Create a new empty `Full`.
+    pub const fn empty() -> Self {
```

Comment:
This is what the `Empty` type is for, no?

Reply:
If client use the `Empty` type for the case to describe empty body where the functionality provided by `Full` is sufficient and do not need to prepare the own Body type, client will need to either box it and erase the type or define the own wrapping type, since `Empty` is a different type from `Full`. Being able to use an empty `Full` type in a const context is useful, as it allows client to define fixed responses that are often used for fallbacks.